core/src/Kokkos_Serial.hpp

```diff
@@ -288,11 +288,16 @@ class TeamPolicyInternal<Kokkos::Serial, Properties...>
   //----------------------------------------
 
   inline int team_size() const { return 1; }
+  inline bool auto_team_size() const { return false; }
+  inline bool auto_vector_length() const { return false; }
```

Comment:
as in the other backends why expose this?

Reply:
impl now